saleor/graphql/product/bulk_mutations/products.py

```diff
@@ -36,6 +36,29 @@ def user_is_allowed(cls, user, _ids):
         return user.has_perm('product.manage_products')
 
 
+class CollectionBulkPublish(BaseBulkMutation):
```

Comment:
What about `ModelBulkPublishMutation`?

Reply:
`Page` still uses this, I decided to remove this mutation in #3974
For me, it is more clear as we see directly `is_published` parameter being used and passed to `bulk_action`.